tests/TC.spec.js

```diff
@@ -0,0 +1,89 @@
+// @ts-check
+const { test } = require('@playwright/test');
+import {
+  LoginPage,
+  ProductsPage,
+  CartPage,
+  CheckoutInformationPage,
+  CheckoutOverviewPage,
+  CheckoutCompletePage,
+  BasePage
+} from '../pages';
+import { standardUser } from '../test_data/users.json';
+import { product1, product2, product3 } from '../test_data/products.json';
+
+test.describe('Checkout Tests', () => {
+  test.beforeEach(async ({ page }) => {
+    const loginPage = new LoginPage(page);
+
+    //Login to a page
+    await loginPage.goto();
+    await loginPage.loginUser(standardUser);
+  });
+
+  test.afterEach(async ({ page }) => {
```

Comment:
Not sure if all feature tests in the `Checkout Tests` suite will require successful order verification at the end.